presto-verifier/src/main/java/com/facebook/presto/verifier/VerifierConfig.java

```diff
@@ -83,6 +83,7 @@
     private boolean shadowWrites = true;
     private String shadowTestTablePrefix = "tmp_verifier_";
     private String shadowControlTablePrefix = "tmp_verifier_";
+    private boolean skipDroppingTemporaryTables = true;
```

Comment:
It's better to have the option be "positive" rather than "negative". With a negative option, setting it to false creates a double negative which is harder to reason about. Thus, this option should be `dropTemporaryTables` so that the result of true/false is more clear.